### ISSUE TITLE
[Crash] Crash when trying to install on low free space

### ISSUE THREAD

**Issue by opk12** [Droid-ify/client#731]
**Describe the bug**
I have little free space. Android auto-deletes the apks before they are installed and Droid-ify crashes.

**To Reproduce**
1. Have little free space.
2. Download an app.
3. Wait a few minutes.
4. Try to install.
5. Droid-ify crashes.

Or

1. Have *very* little free space.
2. Tap to update an app.
3. Droid-ify crashes when it has completed the download and tries to install.

**Smartphone (please complete the following information):**
Droid-ify 0.6.3 on Samsung Android 7.1

**Additional context**
```
FATAL EXCEPTION: DefaultDispatcher-worker-3
Process: com.looker.droidify, PID: 10894
java.io.FileNotFoundException: /data/user/0/com.looker.droidify/cache/releases/d4d4a656343c4d35da3df98d68d24c40a02861dfe1250ffab2b55e5b7d482461.apk (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(FileInputStream.java:146)
	at com.looker.installer.installers.session.SessionInstaller.install(SourceFile:64)
	at com.looker.installer.InstallManager$installer$1.invokeSuspend(SourceFile:226)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(SourceFile)
	at kotlinx.coroutines.DispatchedTask.run(SourceFile)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(SourceFile:96)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@6479853, Dispatchers.Default]
```

**Comment by opk12** [comment]
The free space problem itself could be solved with #3, but I am opening an issue nonetheless because the app should warn and abort instead of crashing.

**Comment by Iamlooker** [comment]
Duplicate of #667
